templates/README.md

```diff
@@ -1,5 +1,6 @@
 <!-- This file was automatically generated by the `build-harness`. Make all changes to `README.yaml` and run `make readme` to rebuild this file. -->
-
+{{ defineDatasource "config" .Env.DATASOURCE }}
```

Comment:
This is what I meant we could do https://github.com/cloudposse/geodesic/blob/master/rootfs/templates/kops/default.yaml#L57